dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -234,6 +234,7 @@ public void testModifyOnGithubRecursive() throws Exception {
         Iterator<GHContent> treeIterator = mock(Iterator.class);
         GHContent content = mock(GHContent.class);
         when(content.isFile()).thenReturn(false, true);
+        when(content.isDirectory()).thenReturn(true, false);
```

Comment:
Code coverage % dropped on the `isFile` usecase not getting covered anymore. You either need to do a `when` on `content.getDownloadUrl()` or flip the `if else` block around to have it be `content.isDirectory() && content.getDownloadUrl() == null`. I think I'd prefer the latter because the test stays simpler (if my theory is correct).